src/pages/rooms/[id]/questions/[questionId]/index.tsx

```diff
@@ -1,15 +1,85 @@
-import React, { ReactElement } from 'react';
+// Was kommt in den Callback der Submitfunktion?
+import React, { ReactElement, SyntheticEvent, useState } from 'react';
 import { NextPageContext } from 'next';
 import api from '../../../../utils/api';
 import getInitialProps from '../../../../utils/get-initial-props';
-import IQuestion from '../../../../../types/question';
+import IQuestion, { QuestionDBModel } from '../../../../../types/question';
+import FreeAnswer from './FreeAnswer';
+import RangeAnswer from './RangeAnswer';
+import FixAnswer from './FixAnswer';
+import { useRouter } from 'next/router';
+import Answer, { AnswerCreate } from '../../../../../types/answer';
+import Link from 'next/link';
 
 interface Props {
-  question: IQuestion;
+  question: QuestionDBModel;
 }
 
 export default function Question({ question }: Props): ReactElement {
-  return <>{JSON.stringify(question)}</>;
+  const [rangeAnswer, setRangeAnswer] = useState();
+  const [fixAnswer, setFixAnswer] = useState('');
+  const [freeAnswer, setFreeAnswer] = useState('');
+  const format = question.answersFormat;
+  const router = useRouter();
+  const questionId = question.id;
+  const roomId = router.query.id;
+
+  const answerFunc = (): AnswerCreate => {
+    if (question.answersFormat === 'range') {
+      return { rangeAnswer };
+    } else if (question.answersFormat === 'fix') {
+      return { fixAnswer };
+    } else if (question.answersFormat === 'free') {
+      return { freeAnswer };
+    }
+  };
+
+  const onSubmit = (e: SyntheticEvent) => {
+    e.preventDefault();
+    if (rangeAnswer !== undefined || fixAnswer !== '' || freeAnswer !== '') {
```

Comment:
```suggestion
    if (rangeAnswer  || fixAnswer || freeAnswer) {
```
should also work, all values are `falsy`
this condition is better placed on the `disabled` attr of submit button